apps/nrm/index.js

```diff
@@ -20,7 +20,27 @@ module.exports = {
       next: '/pv-under-age'
     },
     '/pv-under-age': {
-      next: '/pv-under-age-at-time-of-exploitation'
+      fields: ['pv-under-age'],
```

Comment:
is this from your other branch?

Reply:
Sorry PR'd this one too quickly. Once the other PR is closed these commits will disappear.